docs/image-import.md

```diff
@@ -225,11 +225,38 @@ $ daisy -var:source_image=projectsmy-awesome-projectglobal/images/my-server-impo
 [Daisy] All workflows completed successfully.
 ```
 
-# Limitations and Known Issues
+# Compatibility and Known Limitations
 
-*   Only virtual disks that use Grub as the default boot loader are supported.
 *   Networking: Import workflow sets the interface to DHCP. If that fails, or if there are other interfaces set with firewalls, special routing, VPN's, or other non-standard configurations, networking may fail and while the resulting instance may boot, you may not be able to access it.
 
+Not every VM image will be importable to GCE. Some VMs will have issues after
+import. Below is a list of known compatibility requirements and issues:
+
+### Windows
+| Name | Severity | Description |
+|-|-|-|
+| OS Version | Required | We support the following OS versions: Windows Server 2008 R2, 2012 R2, or 2016; RHEL/CentOS/OEL 6 or 7; Debian 8 or 9; Ubuntu 14.04 or 16.04 |
```

Comment:
If this is Windows only mention only the Windows versions?

Reply:
Done.